common/test/data/uploads/jsChannel.js

```diff
@@ -0,0 +1,535 @@
+var Channel = (function() {
```

Comment:
Please add a comment here saying that this is a copy of `common/static/js/capa/src/jschannel.js`. Even better if you can find a way to use that file directly in the tests, but having a copy here is totally fine, as long as we say where the "source" version is.

Reply:
Interesting, I didn't know that file existed there, I actually got all these files from [the docs](https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/exercises_tools/custom_javascript.html#javascript-input-example-problem-code). I reworked the test to use the file you mentioned, and removed the new one as you suggested.

About the other Nits, I left them as is to match the original source, but I added comments with links to the original sources of the files I included.